src/components/Character/Character.jsx

```diff
@@ -55,29 +65,35 @@ export default function Character(props){
 			aria-roledescription="slide"
 			aria-label={`Character ${index + 1} of ${characterData.length}.`}
 			aria-hidden={ hidden }>
-			<article>
-				<h1 id={`character__${id}`}>
-					{ name }
-				</h1>
-				<h2>
-					{ nickname }
-				</h2>
-				<h3>
-					{ actor }
-				</h3>
-				<p>
-					{ body }
-				</p>
-				<img 
-					src={ src } 
-					alt={ description }
-					style={ mask }
-				/>
-
-				{ prevCharName && RENDER.button(prevCharName, -1) }
-				{ nextCharName && RENDER.button(nextCharName, +1) }
-
+			<article className={ s.container }>
+				<div className={ s.content }>
+					<h1 
+						className={ s.name } 
+						id={`character__${id}`}
+					>
+						{ name }
+					</h1>
+					<h2 className={ s.nickname } >
+						{ nickname }
+					</h2>
+					<h3 className={ s.actor } >
+						{ actor }
+					</h3>
+					<div className={ s.body } > 
+						{ body }
+					</div>	
+				</div>
+				<nav className={ s.controls }>
+					{ prevButton }
+					{ nextButton }
+				</nav>	
 			</article>
+			<HeroImage 
+				idString={ idString }
```

Comment:
```suggestion
				id={ heroId }
```
I don't think an `id` can be anything tother than a string, so maybe a superfluous name here, and `idString` as a value doesn't say much about what it is - so `heroId` works better imho.